source/evp_override.c

```diff
@@ -416,6 +417,14 @@ const EVP_CIPHER *EVP_aes_256_gcm(void) {
     static const EVP_CIPHER cipher = { EVP_AES_256_GCM, 128 };
     return &cipher;
 }
+const EVP_CIPHER *EVP_aes_128_ecb(void) {
+    static const EVP_CIPHER cipher = { EVP_AES_128_ECB, 128 };
+    return &cipher;
+}
+
+void EVP_CIPHER_CTX_init(EVP_CIPHER_CTX *ctx) {
```

Comment:
I'd add a link to the documentation or original implementation of `EVP_CIPHER_CTX_init`. Just to justify we don't have any restrictions on `ctx`.

Reply:
I added what was in the man pages.  Not sure I understand what this is supposed to do though.